### PR TITLE
Fix Conflict in Rel notes

### PR DESCRIPTION
**Pull request by ollypom**
### Proposed changes

Fixes: https://github.com/docker/docker.github.io/issues/9185

### CODE REVIEW

File: engine/release-notes.md [hunk 1]
```diff
@@ -625,34 +632,10 @@ Ubuntu 14.04 "Trusty Tahr" [docker-ce-packaging#255](https://github.com/docker/d
...
-
-2019-07-17
-
-### Runtime
-
-* Masked the secrets updated to the log files when running Docker Engine in debug mode. [CVE-2019-13509](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2019-13509): If a Docker engine is running in debug mode, and `docker stack deploy` is used to redeploy a stack which includes non-external secrets, the logs will contain the secret.
-
-## 18.03.1-ee-9
-
-2019-06-27
```
Comment (adrian-plata) [inline engine/release-notes.md]: Did you mean to delete the 2019-06-27 notes?

**Comment by kolyshkin** [comment]
Thanks, this looks good to me, all the issues that I mentioned are fixed!

File: engine/release-notes.md (hunk 1 above)
Comment (kolyshkin) [reply engine/release-notes.md]: @adrian-plata it was there twice, take a look with more context and you'll see

**Comment by kolyshkin** [comment]
I was using this URL to verify: https://deploy-preview-9186--docsdocker.netlify.com/engine/release-notes/

**Pull request merged**